modules/minimum-required-permissions-ipi-azure.adoc

```diff
@@ -0,0 +1,228 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_azure/installing-azure-account.adoc
+
+
+[id="minimum-required-azure-permissions_{context}"]
+= Required Azure permissions in installer-provisioned infrastructure
+
+When you assign the `Contributor` and `User Access Administrator` roles to the service principal, you automatically grant all the required permissions. To deploy and destroy an {product-title} cluster with installer-provisioned infrastructure, the service principal requires the following permissions.
+
+Following are the permissions required for creating the {product-title} cluster.
+
+.Required permissions for creating authorization resources
+[%collapsible]
+====
+* `Microsoft.Authorization/policies/audit/action`
+* `Microsoft.Authorization/policies/auditIfNotExists/action`
+* `Microsoft.Authorization/roleAssignments/read`
+* `Microsoft.Authorization/roleAssignments/write`
+====
+
+.Required permissions for creating compute resources
+[%collapsible]
+====
+* `Microsoft.Compute/images/read`
+* `Microsoft.Compute/images/write`
+* `Microsoft.Compute/images/delete`
```

Comment:
Following 3 permissions are not required, but not sure if they are needed with marketplace image, will check when marketplace image can be tested.
```
Microsoft.Compute/images/read
Microsoft.Compute/images/write
Microsoft.Compute/images/delete

```

Reply:
Yes, and I don't think they are present in the required list in the google doc, they are present in optional list. 
To clarify: These 3 are optional for IPI, but required for UPI

Reply:
Addressed